docs/elasticsearch.txt

```diff
@@ -94,6 +94,13 @@ After processing `ext`, JanusGraph checks for the following common options.  Jan
 * `index.[X].elasticsearch.ignore-cluster-name`
 * `index.[X].elasticsearch.health-request-timeout`
 
+===== Ingest Pipelines
+If using Elasticsearch 5.0 or higher, a different ingest pipelines can be set for each store.
```

Comment:
"a different ingest pipeline can be set for each store"

I think this section could use a very brief introduction to the ingest pipeline concept. Some users coming to JanusGraph do not have deep knowledge of specific features from the various backends. How and why might somebody use an ingest pipeline with a JanusGraph mixed index?

Reply:
You are right. I change the doc to explain the concept of pipeline and I add an example of use.